spring-cloud-netflix-eureka-client/src/test/java/org/springframework/cloud/netflix/eureka/config/EurekaConfigServerBootstrapConfigurationTests.java

```diff
@@ -163,7 +163,34 @@ public void eurekaConfigServerInstanceProviderCalled() {
 						"eureka.client.enabled=true",
 						"spring.main.sources=" + TestConfigDiscoveryBootstrapConfiguration.class.getName(),
 						"logging.level.org.springframework.cloud.netflix.eureka.config=DEBUG")
-				.run();
+				.run().close();
+		assertThat(output).contains("eurekaConfigServerInstanceProvider finding instances for configserver")
```

Comment:
Please update the year in the license comment and add your full name and surname with `@author` tag in the javadoc of this file as well, since you're changing it.

Reply:
Modified